Makefile

```diff
@@ -95,3 +101,9 @@ install-openscap:
 .PHONY: vagrant-check
 vagrant-check:
 	BOX=$(BOX) sh ./vagrant.sh 
+
+.PHONY: build-skopeo
+build-skopeo:
+	rm -rf $(DESTDIR)/skopeo
+	git clone --depth 1 $(SKOPEO_SRC_REPO) -b $(SKOPEO_SRC_BRANCH) $(DESTDIR)/skopeo
+	make binary --directory $(DESTDIR)/skopeo -d
```

Comment:
Would it be sufficient to just enable updates-testing and install from there? You can add extra repos in the `.redhat-ci.yml` file (see [sample](https://github.com/jlebon/redhat-ci/blob/master/sample.redhat-ci.yml#L65)). For CentOS, you can use the [continuous RPMs](https://wiki.centos.org/SpecialInterestGroup/Atomic/Devel).